server/sonar-server/src/main/java/org/sonar/server/setting/ws/CheckSecretKeyAction.java

```diff
@@ -52,7 +51,7 @@ public void define(WebService.NewController context) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    userSession.checkPermission(SYSTEM_ADMIN);
+    userSession.checkLoggedIn().checkIsRoot();
```

Comment:
description should be fixed, root is now required, not system administration... or is that the name we give to root?